### DISCUSSION TITLE
Does strimzi kafka operator use openssl? Is if safe from currently found vulnerability?

### DISCUSSION THREAD

**Discussion by aldlfkahs** [strimzi/strimzi-kafka-operator#7571] [Q&A] [answered]
Recently, a critical security vulnerability is found at openssl version 3.0.0 ~ 3.0.6. ([link](https://www.openssl.org/news/secadv/20221101.txt))

Does strimzi kafka operator use openssl?
Is it safe to use?

**Comment by scholzj** [comment, marked as answer]
Strimzi does use OpenSSL, but only version 1.1.1 which should not be afected (+ we do not use it to verify certificates anyway). So this should not be a problem for us and it should be safe to use.
